locales/base/translation.json

```diff
@@ -1323,7 +1323,8 @@
     },
     "favoritedDappToast": {
       "message": "Added as a favorite",
-      "labelCTA": "View Favorites"
+      "labelCTA": "View Favorites",
+      "messageWithDappName": "{{dappName}} added as a favorite"
```

Comment:
Do we really want a custom wording for this?

Reply:
😅 i could take it out but wasn't sure if there'd be languages where it'd make more sense for the dappName to be in the middle of the sentence or something